app/utils/tools.py

```diff
@@ -71,26 +71,20 @@ def fuzzy_search_user(
     `query` will be compared against `users` name, firstname and nickname.
     The size of the answer can be limited using `limit` parameter.
 
-    Use RapidFuzz library
+    Use Jellyfish library
     """
 
-    # We can give a dictionary of {object: string used for the comparison} to the extract function
-    # https://maxbachmann.github.io/RapidFuzz/Usage/process.html#extract
-
     # TODO: we may want to cache this object. Its generation may take some time if there is a big user base
```

Comment:
Isn't it the time to do it ?

Reply:
Instead of caching we may explore the list generation in the database
https://docs.sqlalchemy.org/en/20/core/functions.html#sqlalchemy.sql.functions.concat